day_6/exercises/burrito.js

```diff
@@ -16,12 +16,36 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+  add_topping(topping) {
+    this.toppings.push(topping);
+  }
+  remove_topping(topping) {
+    this.toppings.pop(topping);
+  }
+  change_protein(protein) {
```

Comment:
If there is always one protein per burrito, does it need to be in an array? Or would a string suffice?